src/unstract/sdk/index.py

```diff
@@ -416,6 +422,7 @@ def generate_index_key(
         chunk_overlap: str,
         file_path: Optional[str] = None,
         file_hash: Optional[str] = None,
+        fs: FileStorage = FileStorage(provider=FileStorageProvider.Local),
```

Comment:
@gaya3-zipstack QQ on the usage - We have initialed to Local storage here, how do we distinguish between the providers?